File: Selenium/Activities/Activity15.java
package activity;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity15 {

	public static void main(String[] args) {

		WebDriver driver = new ChromeDriver();
		
		driver.get("https://training-support.net/webelements/dynamic-attributes");
		
		System.out.println("Title of the page: "+ driver.getTitle());
		
		WebElement fullName = driver.findElement(By.xpath("//input[@placeholder='Full name']"));
		WebElement email = driver.findElement(By.xpath("//input[contains(@placeholder, 'Email')]"));
		WebElement eventDate = driver.findElement(By.xpath("//input[@type='date']"));
		WebElement details = driver.findElement(By.xpath("//textarea[contains(@name,  'additional-details')]"));
		WebElement submitBtn = driver.findElement(By.cssSelector(".font-bold.svelte-7bqce9"));
		
		fullName.sendKeys("Y Someone");
		email.sendKeys("yone@gmail.com");
		eventDate.sendKeys("17-12-2025");
		details.sendKeys("NA");
		submitBtn.click();
		
		String confirmation = driver.findElement(By.id("action-confirmation")).getText();
		System.out.println("Confirmation messgage: " + confirmation);
		
		driver.quit();

	}

}

File: Selenium/Activities/activity_15.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, timeout=10)

    driver.get("https://training-support.net/webelements/dynamic-attributes")

    print("Title of the page: ", driver.title)

    full_name = driver.find_element(By.XPATH, "//input[@placeholder='Full name']")
    email = driver.find_element(By.XPATH, "//input[contains(@placeholder, 'Email')]")
    event_date = driver.find_element(By.XPATH, "//input[@type='date']")
    details = driver.find_element(By.XPATH, "//textarea[contains(@name,  'additional-details')]")
    submit_btn = driver.find_element(By.CSS_SELECTOR, ".font-bold.svelte-7bqce9")

    full_name.send_keys("Y Someone")
    email.send_keys("yone@gmail.com")
    event_date.click()
    event_date.send_keys("18/12/2025")
    details.send_keys("NA")
    submit_btn.click()

    confirmation = message = wait.until(EC.visibility_of_element_located((By.ID, "action-confirmation"))).text
    print("Confirmation messgage: ", confirmation)
